sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -497,3 +566,23 @@ export enum KnownKeyOperations {
   /** Key operation - encrypt */
   Import = "import"
 }
+
+/** Known values of {@link KeyExportEncryptionAlgorithm} that the service accepts. */
+export enum KnownKeyExportEncryptionAlgorithm {
+  CkmRsaAesKeyWrap = "CKM_RSA_AES_KEY_WRAP",
+  RsaAesKeyWrap256 = "RSA_AES_KEY_WRAP_256",
+  RsaAesKeyWrap384 = "RSA_AES_KEY_WRAP_384"
+}
+
+/* eslint-disable tsdoc/syntax */
```

Comment:
Is this necessary? Isn't there a way to make it look nice while this stays enabled?

Reply:
I tried various ways to make our tsdoc linter happy - it's not happy with the escaping of the newlines on lines 583-585. This is also how they are generated in the generated code so I feel it's reasonable to have them here. I was not able to get newlines rendered correctly without this but I am happy to make an issue to investigate that further

Reply:
I'm convinced to leave it as you have it here :)